go-demo/compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  web:
+    build: .
+    image: mozilla/go-demo
```

Comment:
TODO: change this image name to `cicd-demos/go-demo`, or whatever name is generated by `deploy-actions/docker-build`

My hypothesis is that as long as the image name here matches what's build by `docker-build`, docker compose won't try to rebuild the image and will use the one built by `deploy-actions/docker-build`.

Reply:
https://github.com/mozilla-it/deploy-actions/releases/tag/v5.2.0